server/src/main/java/org/apache/iotdb/db/metadata/schemaregion/SchemaRegion.java

```diff
@@ -1158,6 +1158,11 @@ public List<MeasurementPath> getAllMeasurementByDevicePath(PartialPath devicePat
   // endregion
 
   // region Interfaces and methods for MNode query
+
+  public IMNode getNodeByPath(PartialPath path) throws MetadataException {
+    return mtree.getNodeByPath(path);
+  }
```

Comment:
Remove this since we use specific interfaces for tirgger. It is not suggested to expose mnode, the internal structure of schemaRegion, to the outside module.

Reply:
fixed： rename to getMNodeForTrigger

Reply:
Still exists, please remove.